src/java.desktop/windows/native/libawt/windows/awt_Taskbar.cpp

```diff
@@ -126,9 +126,14 @@ JNIEXPORT void JNICALL Java_sun_awt_windows_WTaskbarPeer_flashWindow
 JNIEXPORT void JNICALL Java_sun_awt_windows_WTaskbarPeer_setOverlayIcon
   (JNIEnv *env, jobject, jlong window, jintArray buf, jint w, jint h)
 {
-    HICON icon = CreateIconFromRaster(env, buf, w, h);
-    m_Taskbar->SetOverlayIcon((HWND)window, icon, NULL);
-    ::DestroyIcon(icon);
+    try {
+        HICON icon = CreateIconFromRaster(env, buf, w, h);
+        m_Taskbar->SetOverlayIcon((HWND)window, icon, NULL);
+        ::DestroyIcon(icon);
+    }
+    catch (const std::bad_alloc&) {
```

Comment:
Please follow java style for catch block too.

```suggestion
    } catch (const std::bad_alloc&) {
```